src/services/APICalls.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const fetchVideos = async (setterFunc) => {
+  try {
+    const response = await axios({
+      method: "GET",
+      url: "/api/videos",
+    });
+    setterFunc(response.data.videos);
+  } catch (e) {
+    console.log("arreeeeeee", e);
```

Comment:
you can update this console.log

Reply:
Thanks a lot, Prathmesh, these slipped my mind.